### PR TITLE
Added unit test case for CMLAtomTypeListTest.java

### PR DESCRIPTION
**Pull request by Gowrishankar04**
#17 - Added unit test case for CMLAtomTypeListTest.java

### CODE REVIEW

**Comment by Gowrishankar04** [comment]
Hi @egonw - I have added unit test case for above mentioned file. Kindly review and provide feedback if any. Thanks!!

File: src/test/java/org/xmlcml/cml/element/main/CMLAtomTypeListTest.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
+package org.xmlcml.cml.element.main;
```
Comment (egonw) [inline src/test/java/org/xmlcml/cml/element/main/CMLAtomTypeListTest.java:21]: Please add a copyright/license statement. See for example the `CMLArrayListTest.java` file in this repository.

File: src/test/java/org/xmlcml/cml/element/main/CMLAtomTypeListTest.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+public class CMLAtomTypeListTest {
+
+    @Test
+    public void testCMLAtomTypeList() {
+        CMLAtomTypeList cmlAtomTypeList = new CMLAtomTypeList();
+        Assert.assertNotNull("constructor: ", cmlAtomTypeList);
+        cmlAtomTypeList.setAttribute("atom", "atomTypeList");
+        CMLAtomTypeList cmlAtomTypeList1 = new CMLAtomTypeList(cmlAtomTypeList);
+        Assert.assertEquals("class should be CMLAtomTypeList: ", cmlAtomTypeList.getAttributeValue("atom"),
```
Comment (egonw) [inline src/test/java/org/xmlcml/cml/element/main/CMLAtomTypeListTest.java]: The error message does not seem to match the test. Please update.

**Review by egonw — CHANGES_REQUESTED** [review]
Two comments, otherwise looks good.

**Comment by Gowrishankar04** [comment]
@egonw  - Sure! let me make these changes and include other files as well. Thanks!!

**Comment by Gowrishankar04** [comment]
@egonw - Added unit test case for the below files:

CMLAtomTypeList.java, CMLBand.java, CMLBandList.java, CMLAxis.java, CMLXAxis.java, CMLYAxis.java.

Kindly review and merge the PR.

Thanks!!

File: src/test/java/org/xmlcml/cml/element/main/CMLBandListTest.java [hunk 3]
```diff
@@ -0,0 +1,62 @@
+/**
+ *    Copyright 2011 Peter Murray-Rust et. al.
```
Comment (egonw) [inline src/test/java/org/xmlcml/cml/element/main/CMLBandListTest.java]: Hi, the copyright statement format is as followed. First, in most countries the author or authors of a patch is/are the copyright holder. If you copied and started with code from someone else, that copyright is indeed often kept. But the copyright statement should also have the year of the new code (2023) and the name of the author. Your name, if you write the code all by yourself.

Please update the copyright statements. This is important to us to be compatible with projects like Debian, Red Hat, etc.

**Review by egonw — CHANGES_REQUESTED** [review]

**Pull request merged**